plugins/modules/database/postgresql/postgresql_info.py

```diff
@@ -550,6 +555,7 @@ def __init__(self, module, db_conn_obj):
         self.cursor = db_conn_obj.connect()
         self.pg_info = {
             "version": {},
+            "in_recovery": None,
```

Comment:
I'm not sure, but if you use `type: bool` for `in_recovery`, then it will be confusing if you return `none` for that setting, or? I would assume that `false` will be the "default" return value and only if recovery mode is on, it will return `true`.
Please speak up if I oversee something here :)

Reply:
@tcraxs thanks for the question! We should return something anyway, even if users filter this value when they don't want to see it. So, `None` is used only for that (like a filler), there's no default value. I don't think there are a lot of people who want to use `filter: !in recovery` but anyway